### DISCUSSION TITLE
Markdown Editor: Show simple formatting while editting

### DISCUSSION THREAD

**Discussion by ghost** [GitJournal/GitJournal#948] [Feature Requests]
It would be really neat if there was a way to show Markdown changes in the editor. For example, if you typed ``*Italic*``, the word would be instantly italicized the moment you added the closing ``*``, instead of having to go to view mode to see your Markdown rendered.

Is something like this in the works?

**Comment by vHanda** [comment]
This is something I had explored long back, but I never finished it up. It certainly would be nice to have.
